### DISCUSSION TITLE
Pulsante continuo

### DISCUSSION THREAD

**Discussion by lucaquartieri** [indomus/forum#3090] [HUB: Home Assistant]
Avrei necessità di un bottone in una card che finche è premuto mantenga attivo uno switch, appena tolgo il dito dal bottone disattiva lo switch.
Non ho trovato nulla di funzionante, possibile che nessuno ha mai avuto questa necessità e HA non permetta di farlo?

**Comment by angio89** [comment]
Se non sbaglio è possibile assegnare delle azioni alla **pressione corta** ed alla **pressione prolungata** della card, ma non un'azione continua o comunque per adesso personalmente non ho mai avuto necessità di questo tipo di azione.
